lib/scholar/metrics.ex

```diff
@@ -118,13 +124,15 @@ defmodule Scholar.Metrics do
     true_positives = binary_true_positives(y_true, y_pred)
     false_negatives = binary_false_negatives(y_true, y_pred)
 
-    true_positives / (false_negatives + true_positives + 1.0e-16)
+    safe_division(true_positives, false_negatives + true_positives)
   end
 
   @doc ~S"""
   Computes the recall of the given predictions with respect to
   the given targets for multi-class classification problems.
 
+  If the sum of true positive and false positive examples equals to 0, then the result is set to 0.
```

Comment:
```suggestion
  If the sum of true positives and false negatives is 0, then the
  result is 0 to avoid zero division.
```

Reply:
Ok, I'll change the other docs accordingly.